doc/source/whatsnew/v0.24.0.txt

```diff
@@ -673,7 +673,7 @@ I/O
 
 - :func:`read_html()` no longer ignores all-whitespace ``<tr>`` within ``<thead>`` when considering the ``skiprows`` and ``header`` arguments. Previously, users had to decrease their ``header`` and ``skiprows`` values on such tables to work around the issue. (:issue:`21641`)
 - :func:`read_excel()` will correctly show the deprecation warning for previously deprecated ``sheetname`` (:issue:`17994`)
--
+- Bug in :meth:`detect_client_encoding` where potential ``IOError`` goes unhandled when importing in a mod_wsgi process due to restricted access to stdout. (:issue:`21552`)
```

Comment:
Maybe move to 0.23.5? @jreback

Reply:
Is this a regression?  If not, I would probably stay with `0.24.0` unless there is a compelling reason to backport-patch this bug.